supported-version/src/versions/mage-os/individual.json

```diff
@@ -95,6 +95,20 @@
         "nginx": "nginx:1.26",
         "os": "ubuntu-latest",
         "release": "2025-02-12T00:00:00+0000",
-        "eol": "2027-02-14T00:00:00+0000"
+        "eol": "2025-04-07T00:00:00+0000"
+    },
+    "mage-os/project-community-edition:1.1.0": {
+        "magento": "mage-os/project-community-edition:1.1.0",
+        "php": 8.4,
```

Comment:
shouldn't this be 8.3? 8.3 is the requirement for 2.4.8 https://experienceleague.adobe.com/en/docs/commerce-operations/installation-guide/system-requirements

Reply:
8.3 and 8.4 are both supported. Should these files encode the minimum, maximum, or median supported version? I think I assumed latest compatible versions available for Warden. (https://hub.docker.com/u/wardenenv)

Reply:
I've no idea 😰 I think @Vinai is the only one who knows that